app/config/config.dist.json

```diff
@@ -147,5 +147,22 @@
 			"pageTitles": "pageTitles.json",
 			"membershipTypes": "membershipTypes.json"
 		}
+	},
+	"payment-types": {
+		"BEZ": {
+			"enabled": true
+		},
+		"UEB": {
+			"enabled": true
+		},
+		"MCP": {
+			"enabled": true
+		},
+		"PPL": {
+			"enabled": true
+		},
+		"SUB": {
+			"enabled": true
+		}
```

Comment:
Looking at the whole "paypal for memberships" issue where we do a "feature toggle" by delaying template changes (#901,#911), maybe we should do the feature toggle by domain (membership and donation).